addons/website_event/models/event_event.py

```diff
@@ -507,6 +507,14 @@ def _track_subtype(self, init_values):
             return self.env.ref('website_event.mt_event_unpublished', raise_if_not_found=False)
         return super(Event, self)._track_subtype(init_values)
 
+    def _get_external_description(self):
+        # Adding the URL of the event into the description
+        self.ensure_one()
+        max_length = 1900
+        event_url = f'<a href="{self.get_base_url()}{self.website_url}/register">{self.name}</a>'
```

Comment:
A bit better to use "url_join" to combine the various path.

Reply:
Hello @awa-odoo 
rather than joining the various path I use this field `event_register_url` which contains the proper URL that I am making if that's okay to you